packages/nuxt-module/plugins/domain.js

```diff
@@ -50,8 +51,22 @@ export default async ({ app, route, req }, inject) => {
         hostname = window.location.hostname;
         pathname = window.location.pathname;
       } else if (!process.static) {
-        const detectedHost =
-          req.headers["x-forwarded-host"] || req.headers.host;
+        // for ALB healtchecks we simply take the first available domain, so we dont generate errors
+        if (
+          req.headers["user-agent"].includes("ELB-HealthChecker") &&
```

Comment:
Please use optional chaining in case of user-agent header is somehow empty.
```suggestion
          req.headers["user-agent"]?.includes("ELB-HealthChecker") &&
```

Reply:
done